src/app/password-bars/password-bars.component.ts

```diff
@@ -0,0 +1,89 @@
+import { Component, Input, SimpleChange, Output, EventEmitter } from '@angular/core';
+
+enum Strength {
+  eightSymbols = 1,
+  easy = 2,
+  medium = 3,
+  strong = 4
+}
+
+@Component({
+  selector: 'app-password-bars',
+  templateUrl: './password-bars.component.html',
+  styleUrls: ['./password-bars.component.scss']
+})
+export class PasswordBarsComponent {
+  @Input() public passwordToCheck: string;
```

Comment:
```suggestion
  @Input() passwordToCheck: string;
```
use `public` everywhere or don't use at all